keyring-util/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "keyring-util"
```

Comment:
That tool already exist and this is definitively a new different one, which fills its role, but not necessarily in the same way. I think this more as a debugging tool than as part of the desktop  e.g. one could for example allow giving the tool a keyring file and use that instead of the portal. Apps and other services can just use oo7 or libsecret.

Reply:
I think it's confusing to use "keyring" as a name. That's already the name of GNOME's secret service implementation and the name of two different file formats used in two different contexts. If you are only accessing keyring files with it okay, but not if it's using secret service or even only using secret service.

Reply:
I am open to suggestion. Note that this util interacts with secrets, secret service, and keyrings, it will be hard to find a perfect name.

Reply:
As this is not something we will be releasing any time soon, I would not block on it and we can open a separate issue to come up with a different name.